cmd/commands/server.go

```diff
@@ -71,5 +73,6 @@ func NewServerCommand() *cobra.Command {
 	command.Flags().StringVar(&baseHref, "base-href", sharedutil.LookupEnvStringOr("NUMAFLOW_SERVER_BASE_HREF", "/"), "Base href for Numaflow server, defaults to '/'.")
 	command.Flags().BoolVar(&disableAuth, "disable-auth", sharedutil.LookupEnvBoolOr("NUMAFLOW_SERVER_DISABLE_AUTH", false), "Whether to disable authentication and authorization, defaults to false.")
 	command.Flags().StringVar(&serverAddr, "server-addr", sharedutil.LookupEnvStringOr("NUMAFLOW_SERVER_ADDRESS", "https://localhost:8443"), "The external address of the Numaflow server.")
+	command.Flags().StringSliceVar(&corsAllowedOrigins, "cors-allowed-origins", strings.Split(sharedutil.LookupEnvStringOr("NUMAFLOW_SERVER_CORS_ALLOWED_ORIGINS", ""), ","), "The default values for allowed cors AllowOrigins header field")
```

Comment:
```suggestion
	command.Flags().StringSliceVar(&corsAllowedOrigins, "cors-allowed-origins", strings.Split(sharedutil.LookupEnvStringOr("NUMAFLOW_SERVER_CORS_ALLOWED_ORIGINS", ""), ","), "The values for allowed cors AllowOrigins header field, separated by comma")
```

Reply:
also if the default value for origin header is `""` it will give - `bad origin: origins must contain '*' or include http://,https://`, let's make sure to give a default origin